.github/workflows/tests.yaml

```diff
@@ -45,15 +43,15 @@ jobs:
 
       - name: Run tests
         run: |
-          unset GITHUB_ACTIONS
-          vendor/bin/phpunit --colors=always --coverage-clover build/logs/clover.xml
+          (unset GITHUB_ACTIONS
+          vendor/bin/phpunit --colors=always --coverage-clover build/logs/clover.xml)
 
       - name: Submit coverage to Coveralls
         env:
           COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NO_COLOR: 'true'
         run: |
-          export GITHUB_ACTIONS=true # Set it back so that php-coveralls properly detects Github Actions
-          composer global require twinh/php-coveralls --ignore-platform-req=php
+          composer global require php-coveralls/php-coveralls:^2.4.2 --ignore-platform-req=php
```

Comment:
Is `--ignore-platform-req=php` still needed? I believe `php-coveralls` now contains PHP 8 support.

Reply:
Indeed it isn't. And this person was the one who added it 🤦